docs/user/exercises/quiz.rst

```diff
@@ -1,2 +1,440 @@
+
+                - Explanations [exp]
+
+                - Hints [hint]
+
+            - Text editor: The quiz statement can be developed with options, hints and explanations.
+
+            - Preview View: Enables the student view.
+
+        - **Add Drag-And-Drop Question**
+
+            This kind of question is composed by a problem statement, a background image, and drag and drop options.
+
+            .. figure:: quiz/drag-and-drop-question.png
+                :align: center
+
+            - Short question title: Provide a short title.
+
+            - Points: Assign the value points for this question.
+
+            - Scoring type:
+
+                - All or nothing
+
+                - Proportional with Penalty
+
+                - Proportional without Penalty
+
+            - Present Drag Items in Random order.
+
+            - Delete icon: To delete the current question.
+
+            - Edit View: Enables the text editor to write the question statement with explanations and hints.
+
+            - Edit bar: When the edit view is enabled, the format bar provides:
+
+                - Style to the statement text
+
+                - Explanations [exp]
+
+                - Hints [hint]
+
+            - Text editor: The quiz statement can be developed with hints and explanations.
+
+            - Upload Background: To select and upload the background from the PC files to drag and drop the options over it.
+
+            - Add Drag Items:
+
+                - Text items: Type the options.
+
+                - Image items: Can be uploaded from the PC files.
+
+            - Preview View: Enables the student view.
+
+        - **Add Short-Answer Question**
+
+            This kind of question is composed by a statement and spots to fill them out by typing the answers.
+
+            .. figure:: quiz/short-answer-question.png
+                :align: center
+
+            - Short question title: Provide a short title.
+
+            - Points: Assign the value points for this question.
+
+            - Scoring type:
+
+                - All or nothing
+
+                - Proportional with Penalty
+
+                - Proportional without Penalty
+
+            - Match Letter Case
+
+            - Match Answers Exactly: This option moves the match slider to 100%.
+
+            - Delete icon: To delete the current question.
+
+            - Add Spot Option: To add the spot between the text to be filled out.
+
+            - Add Answer Option: To provide the answer for each spot.
+
+            - Text editor: The quiz statement can be developed with the spots and options.
+
+            - Text view button: Enables the text editor to write and edit the question statement.
+
+            - Preview View button: Enables the student view.
+
+        - **Add Existing Questions**
+
+            This is an option that allows to import existing questions from other quiz exercises, courses, exams and files.
```

Comment:
that allows importing
(import should be in gerund form when followed by allows)